sycl/test/lit.site.cfg.py.in

```diff
@@ -26,7 +26,7 @@ config.test_include_path = "@TEST_INCLUDE_PATH@"
 config.llvm_enable_projects = "@LLVM_ENABLE_PROJECTS@"
 
 config.sycl_threads_lib = '@SYCL_THREADS_LIB@'
-config.sycl_use_libcxx = 'ON' if bool('@LLVM_LIBCXX_USED@') else 'OFF'
+config.sycl_use_libcxx = 'ON' if @LLVM_LIBCXX_USED@ else 'OFF'
```

Comment:
`bool('0')` is actually true in python, since the value of the CMake var should be `1` or `0` just use the literal value.